### ISSUE TITLE
get Location of detected image

### ISSUE THREAD

**Issue by lalitchoudhary10007** [AndroidArena/CurrencyDetectorAndroid#1]
Hi, 
First of all thanks for your great tutorial.
I need location(RectF) of detected images...now i am getting 0 in every because we have passed null in below line in Classifier class

pq.add(new Recognition("" + entry.getKey(), entry.getKey(), entry.getValue(), null)); 

can you please help me with this...i already tried many things but no luck.

I will appreciate if you can give some hint or idea.

basically i need a border around the detected image.

Thanks
